docs/production-candig.md

```diff
@@ -0,0 +1,177 @@
+
+File requirements:
+- Name the file siteLogo.png
+- The image should be square and will be set to 34x34 pixels
+- The image format must be PNG
+
+If the portal is already running, copy the logo into the Docker container using this command:
+
+```bash
+ docker cp Your_images_path/siteLogo.png candigv2_candig-data-portal_1:/app/candig-data-portal/src/assets/images/users
+ ```
+ Otherwise:
+
+ ```bash
+ cp your_image_path/siteLogo.png CanDIGv2/lib/candig-data-portal/candig-data-portal/src/assets/images/users/siteLogo.png
+ ```
+
+## Changing the default site admin
+
+When CanDIG is initially deployed, a `site_admin` user will be created by default. The username and password for this user can be found in the `env.sh` file. It is important to change this default to a real user who should have site administration privileges. 
+
+1. Login to the data portal with the credentials you wish to make a site administrator to ensure the user can login successfully
+
+2. Get a site admin token using the default site admin user:
+
+```bash
+source env.sh
+```
+
+```bash
+CURL_OUTPUT=$(curl -s --request POST \
+  --url $KEYCLOAK_PUBLIC_URL'/auth/realms/candig/protocol/openid-connect/token' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data grant_type=password \
+  --data client_id=$CANDIG_CLIENT_ID \
+  --data client_secret=$CANDIG_CLIENT_SECRET \
+  --data username=$CANDIG_SITE_ADMIN_USER \
+  --data password=$CANDIG_SITE_ADMIN_PASSWORD \
+  --data scope=openid)
+```
+
+```bash
+export TOKEN=$(echo $CURL_OUTPUT | grep -Eo 'access_token":"[a-zA-Z0-9._\-]+' | cut -d '"' -f3)
+```
+
+3. Set the role of the real user to a site admin with the following curl command:
+
+```bash
+curl -X POST $CANDIG_URL'/ingest/site-role/admin/email/<YOUR-EMAIL-ADDRESS>' -H 'Authorization: Bearer '$TOKEN
+```
+
+4. Check the role assignment was successful by verifying the following command returns `True`:
+
+```bash
+curl -X GET $CANDIG_URL'/ingest/site-role/admin/email/<YOUR-EMAIL-ADDRESS>' -H 'Authorization: Bearer '$TOKEN
+```
+
+5. Delete the default site admin user using your new real user site admin token
+
+```bash
+CURL_OUTPUT=$(curl -s --request POST \
+  --url $KEYCLOAK_PUBLIC_URL'/auth/realms/candig/protocol/openid-connect/token' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data grant_type=password \
+  --data client_id=$CANDIG_CLIENT_ID \
+  --data client_secret=$CANDIG_CLIENT_SECRET \
+  --data username=<YOUR-EMAIL-ADDRESS> \
+  --data password=<YOUR-PASSWORD> \
+  --data scope=openid)
+```
+
+```bash
+export TOKEN=$(echo $CURL_OUTPUT | grep -Eo 'access_token":"[a-zA-Z0-9._\-]+' | cut -d '"' -f3)
+```
+
+```bash
+curl -X GET $CANDIG_URL'/ingest/site-role/admin/email/site_admin@test.ca' -H 'Authorization: Bearer '$TOKEN
+```
+
+
+## Connecting Keycloak to institutional LDAP
```

Comment:
Is there any generic information we could add here for how people can connect their LDAP to keycloak? @OrdiNeu @SonQBChau ?